example/output/ts/Address.ts

```diff
@@ -0,0 +1,11 @@
+export const Address = {
```

Comment:
are we sure that we want to commit all these examples?

The example folder is also configured in the `files` section of the package JSON, making this a bit heavier than needed

Reply:
For other converters, the examples are maintained as well, that's why I have also put the examples of what is expected after conversion. It has nothing to do with code and logic, just added to keep the standards in sync.

Reply:
OK but still IMO the examples should not be part of the published package.
@simoneb wdyt?

Reply:
the examples [are already](https://github.com/nearform/openapi-transformer-toolkit/blob/master/package.json#L23) part of the published package, I'm not sure why it was decided that way. I don't have a strong opinion about this, just pointing out that this is not a new behavior introduced in this PR.